### ISSUE TITLE
Accessibility: No audio description or transcript

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1050]
## Issue
The video player has visible text content that is not announced to screen reader users via an audio description track or by an adjacent transcript

## Expected Behavior
- [WCAG 2.2 A 1.2.3 Audio Description or Media Alternative (Prerecorded)](https://www.w3.org/WAI/WCAG22/Understanding/audio-description-or-media-alternative-prerecorded.html) requires that an audio track whcih describes important video content or a text based transcript is provided
- An audio description track may be provided dependent on the media player or a text transcript
- Screen reader users expect to have access to important content in a video (aside from dialogue) that sighted users get by default
## Acceptance Criteria
- [ ] Research the video player and analyze the video content to determine if an Audio Description track or transcript is more beneficial
- [ ] If an audio track is possible, record a dialogue track which describes the video and amend it to the video, otherwise create a text transcript (which can be hidden offscreen)
- [ ] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. 
- [ ] Ensure the important video content is announced as part of the audio track or text transcript

_Linked to tracking issue: #456_
